upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.9.yaml.template

```diff
@@ -0,0 +1,543 @@
+# Canal Version v3.2.3
+# https://docs.projectcalico.org/v3.2/releases#v3.2.3
+# This manifest includes the following component versions:
+#   calico/node:v3.2.3
+#   calico/cni:v3.2.3
+#   coreos/flannel:v0.9.1
```

Comment:
Just needs a small comment update here now to match the version being used, otherwise LG thanks :)